ssm/ssm2.py

```diff
@@ -333,35 +361,14 @@ def _initialise_connection(self, host, port):
                                   "wasn't found.")
             log.info('Connecting using SSL...')
 
-            try:
-                # Compatible with stomp.py >= 3.0.4
-                self._conn = stomp.Connection([(host, port)],
-                                              use_ssl=self._use_ssl,
-                                              user=self._user,
-                                              passcode=self._pwd,
-                                              ssl_key_file=self._key,
-                                              ssl_cert_file=self._cert,
-                                              ssl_version=ssl.PROTOCOL_SSLv23)
-            except TypeError:
-                # For stomp.py <= 3.0.3, override ssl.PROTOCOL_SSLv3 and then
-                # try to set up the connection again below.
-                ssl.PROTOCOL_SSLv3 = ssl.PROTOCOL_SSLv23
+        # _conn will use the default SSL version specified by stomp.py
```

Comment:
That looks to be `ssl.PROTOCOL_TLSv1`. Has that been tested on both SL6 and 7?

Reply:
yes